src/commands/game/songcheck.js

```diff
@@ -1,59 +1,61 @@
+'use strict';
+
 // Dependencies
 const config = require('../../../config');
 const Discord = require('discord.js');
 
-module.exports = {
-  config: {
-    name: 'songcheck',
-    description: 'Send a message and ask participants to voice ownership of the given song',
-    options: [{
```

Comment:
Formatting multi-line objects inside an array can be done usually in these two ways:
```javascript
// option 1
let arr = [{
  key: 'value',
  key2: 'value2',
}, {
  key: 'value',
  key2: 'value2',
}]

// option 2
let arr = [
  {
    key: 'value',
    key2: 'value2',
  },
  {
    key: 'value',
    key2: 'value2',
  },
]
```
For now I chose option 1, but you're more than welcome to change to option 2

Reply:
Most likely will stick with option one as it keeps the bracket start lines consitent based on how I position the first bracket.